worker/caasoperator/caasoperator.go

```diff
@@ -286,7 +286,7 @@ func (op *caasOperator) init() (*LocalState, error) {
 
 func (op *caasOperator) loop() (err error) {
 	defer func() {
-		if errors.IsNotFound(err) {
+		if errors.IsNotFound(errors.Cause(err)) {
```

Comment:
don't need this - IsNotFound uses Cause() internally

Reply:
yeah, removed